DirectEmployers/MyJobs
# [PD 2855] metric sums

Minor Fix to the metric sums taking the numbers in parentheses out of the table at the bottom

To Test:

Pull down repo and check the metric sums at the bottom of the table. The numbers in parentheses from before should be gone.
